tests/components/dynalite/test_init.py

```diff
@@ -11,6 +11,21 @@
 from tests.common import mock_coro
 
 
+async def test_empty_config():
+    """Test with an empty config."""
+    hass = Mock()
+    hass.data = {}
+    config = {}
+    mock_conf_host = Mock(return_value=[])
+    with patch(
+        "homeassistant.components.dynalite.__init__.configured_hosts", mock_conf_host
+    ):
+        assert await async_setup(hass, config)
```

Comment:
Don't call integration methods directly. Use the core to set up the tests and then assert the state from the core interfaces. Eg if we want to assert the state of an entity we get the state from the core state machine, `hass.states`.

Use `async_setup_component` from homeassistant/setup.py, to set up the integration.

Reply:
fixed